src/xia2/Modules/MultiCrystal/__init__.py

```diff
@@ -27,10 +27,13 @@
     """\
 unit_cell = None
   .type = unit_cell
+  .short_caption = "unit cell"
```

Comment:
I note that elsewhere the first character of the `short_caption` is capitalised
```suggestion
  .short_caption = "Unit cell"
```